runtime/auth/aws-signing-common/common/src/aws/smithy/kotlin/runtime/auth/awssigning/HashSpecification.kt

```diff
@@ -39,6 +39,11 @@ public sealed class HashSpecification {
      */
     public object StreamingAws4HmacSha256PayloadWithTrailers : HashLiteral("STREAMING-AWS4-HMAC-SHA256-PAYLOAD-TRAILER")
 
+    /**
+     * The hash value used for streaming unsigned requests with trailers
+     */
+    public object StreamingUnsignedPayloadWithTrailers : HashLiteral("STREAMING-UNSIGNED-PAYLOAD-TRAILER")
+
     /**
      * The hash value should indicate ???
```

Comment:
not related to your PR but this comment could be updated, the sha256-events hash literal indicates an event stream (e.g. `vnd.amazon.event-stream` content type)